assets/batch_score/components/driver/src/batch_score/utils/common.py

```diff
@@ -70,62 +66,11 @@ def convert_result_list(results: "list[ScoringResult]", batch_size_per_request:
     return list(map(__stringify_output, output_list))
 
 
-def convert_to_list(
```

Comment:
Why is the removed?

Reply:
Because it is moved to separate versions of input handler. Part of this PR - https://dev.azure.com/msdata/Vienna/_git/batch-score/pullrequest/1303838?_a=files&path=/driver/src/batch_score/utils/common.py